src/main/resources/org/jenkinsci/plugins/docker/traceability/DockerTraceabilityPluginConfiguration/help-createImageFingerprints.html

```diff
@@ -0,0 +1,14 @@
+<div>
+  By default, the plugin registers traceability reports for images, which have
```

Comment:
BTW there should be no comma here in English. [Details](http://en.wikipedia.org/wiki/Dependent_clause#English_punctuation) (I know Czech requires one in this construction, maybe Russian is the same?)